File: 在排序数组中查找元素的第一个和最后一个位置/C++/original.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        int n = nums.size();
        vector<int> temp;
        for(int i=0; i<n; ++i){
            if(nums[i] == target) temp.push_back(i);
        }
        if(temp.empty()) return {-1,-1};
        else return {temp[0], temp[temp.size()-1]};
    }
};
// 思路和python一样，不过数组的表达差别很大。

File: 初级算法/在排序数组中查找元素的第一个和最后一个位置/python3/original.py
class Solution:
    def searchRange(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        思路：把数组遍历一遍，把target的所有元素位置存在temp中，
        返回temp的首尾元素，如果temp为空，返回[-1，-1]。
        """
        n = len(nums)
        temp = []
        for i in range(n):
            if nums[i] == target:
                temp.append(i)
        if temp==[]:
            return [-1, -1]
        else:
            return [temp[0], temp[-1]]